frb_example/with_flutter/lib/bridge_generated.io.dart

```diff
@@ -12,7 +12,26 @@ import 'package:meta/meta.dart';
 import 'dart:ffi' as ffi;
 
 class FlutterRustBridgeExamplePlatform extends FlutterRustBridgeBase<FlutterRustBridgeExampleWire> {
-  FlutterRustBridgeExamplePlatform(ffi.DynamicLibrary dylib) : super(FlutterRustBridgeExampleWire(dylib));
+  final _port = RawReceivePort();
+  NativePortType get port => _port.sendPort.nativePort;
+  FlutterRustBridgeExamplePlatform(ffi.DynamicLibrary dylib) : super(FlutterRustBridgeExampleWire(dylib)) {
+    var initResult =
+        dylib.lookupFunction<ffi.IntPtr Function(ffi.Pointer<ffi.Void>), int Function(ffi.Pointer<ffi.Void>)>(
+            'init_dart_api_dl')(ffi.NativeApi.initializeApiDLData);
+
+    if (initResult != 0) {
+      throw 'Failed to initialize Dart API. Code: $initResult';
+    }
+
+    _port.handler = (response) {
+      inner.drop_DartOpaqueFlutterRustBridgeExample(response);
+    };
+    dylib.lookupFunction<ffi.IntPtr Function(ffi.Pointer<ffi.Void>), int Function(ffi.Pointer<ffi.Void>)>(
+        'init_dart_api_dl')(ffi.NativeApi.initializeApiDLData);
```

Comment:
We need to make "init_dart_api_dl" as well as "create the port" lazily loaded. In other words, do not do it, unless we really want to create a Dart opaque. Otherwise, for all users who do not use Dart opaque, they are paying extra price (and risking possible bugs)